tools/clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -4443,7 +4443,6 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
                     options::OPT_fno_gnu_inline_asm, true))
     CmdArgs.push_back("-fno-gnu-inline-asm");
 
-#if 0
   // XXX Nyuzi: disable loop vectorizer by default, because it does not
```

Comment:
This comment is no longer true, since the loop vectorizer should now work. I think it can just be eliminated.